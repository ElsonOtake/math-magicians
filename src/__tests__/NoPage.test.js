import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import NoPage from '../components/NoPage';

describe('Tests for NoPage component', () => {
  render(<NoPage />);
  test('Check for the text 404 on screen', () => {
    expect(screen.queryByText(/404/)).toBeTruthy();
  });
  test('Check for the snapshot of debug', () => {
    expect(screen.debug()).toMatchSnapshot();
  });
});
