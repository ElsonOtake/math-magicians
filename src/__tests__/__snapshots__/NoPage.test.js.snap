// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for NoPage component Check for the snapshot of debug 1`] = `undefined`;
